constraints.txt

```diff
@@ -0,0 +1 @@
+ansible==2.9.15
```

Comment:
This is to ensure the latest 2.9 is used. Since Ansible is an indirect dependency (by Molecule), apparently this is how you influence it in Python. :D